index.js

```diff
@@ -1,21 +1,37 @@
 /* @flow */
-import fs from 'react-native-fs';
+import fs from "react-native-fs";
+import { Platform } from "react-native";
 
 export const DocumentDir = fs.DocumentDirectoryPath;
 export const CacheDir = fs.CachesDirectoryPath;
+const OSPathSeparator = Platform.select({
+  ios: "/",
+  android: "/",
+  windows: "\\",
+});
 
-const resolvePath = (...paths: Array<string>) =>
-  '/' +
-  paths
-    .join('/')
-    .split('/')
-    .filter(part => part && part !== '.')
-    .join('/');
+const resolvePath = (...paths: Array<string>) => {
+  if (Platform.OS === "windows") {
+    return paths
+      .join(OSPathSeparator)
+      .split(OSPathSeparator)
+      .filter((part) => part && part !== ".")
+      .join(OSPathSeparator);
+  }
```

Comment:
if you're able to use `path` then I think this is the same as `path.normalize` with a trailing separator?